File: lesson3/python3/homework/homework3_2.py
#!/usr/bin/python3

# Напишите программу, каторая найдет максимум и минимум из пяти случанных чисел
# вводимых пользователем, и вывести на экран.
print("Введите пять разных значений, что-бы найти максимум и минимум")
value_a = int(input("value_a => "))
value_b = int(input("value_b => "))
value_c = int(input("value_c => "))
value_d = int(input("value_d => "))
value_e = int(input("value_e => "))

print("Максимальное значение")
if value_a > value_b and value_a > value_c:
    print("value_a =>", value_a)
elif value_b > value_c and value_b > value_d:
    print("value_b =>", value_b)
elif value_c > value_d and value_c > value_e:
    print("value_c =>", value_c)
elif value_d > value_e and value_d > value_a:
    print("value_d =>", value_d)
else:
    print("value_e =>", value_e)

print("Минимальное значение")
if value_a < value_b and value_a < value_c:
    print("value_a =>", value_a)
elif value_b < value_c and value_b < value_d:
    print("value_b =>", value_b)
elif value_c < value_d and value_c < value_e:
    print("value_c =>", value_c)
elif value_d < value_e and value_d < value_a:
    print("value_d =>", value_d)
else:
    print("value_e =>", value_e)

File: lesson3/c_c++/homework/homework3_2.c
#include <stdio.h>

int main() {
    int value_a, value_b, value_c, value_d, value_e;

    printf("Введите пять разных значений, что-бы найти максимум и минимум\n");
    printf("value_a => ");
    scanf("%d", &value_a);
    printf("value_b => ");
    scanf("%d", &value_b);
    printf("value_c => ");
    scanf("%d", &value_c);
    printf("value_d => ");
    scanf("%d", &value_d);
    printf("value_e => ");
    scanf("%d", &value_e);

    printf("Максимальное значение\n");
    if (value_a > value_b && value_a > value_c) {
        printf("value_a => %d\n", value_a);
    } else if (value_b > value_c && value_b > value_d) {
        printf("value_b => %d\n", value_b);
    } else if (value_c > value_d && value_c > value_e) {
        printf("value_c => %d\n", value_c);
    } else if (value_d > value_e && value_d > value_a) {
        printf("value_d => %d\n", value_d);
    } else {
        printf("value_e => %d\n", value_e);
    }

    printf("Минимальное значение\n");
    if (value_a < value_b && value_a < value_c) {
        printf("value_a => %d\n", value_a);
    } else if (value_b < value_c && value_b < value_d) {
        printf("value_b => %d\n", value_b);
    } else if (value_c < value_d && value_c < value_e ) {
        printf("value_c => %d\n", value_c);
    } else if (value_d < value_e && value_d < value_a) {
        printf("value_d => %d\n", value_d);
    } else {
        printf("value_e => %d\n", value_e);
    }

    return 0;
}
